ptvsd/_util.py

```diff
@@ -0,0 +1,245 @@
+from __future__ import print_function
+
+import contextlib
+import os
+import threading
+import sys
+
+
+DEBUG = False
+if os.environ.get('PTVSD_DEBUG', ''):
+    DEBUG = True
+
+
+def debug(*msg, **kwargs):
+    if not DEBUG:
+        return
+    tb = kwargs.pop('tb', False)
+    assert not kwargs
+    if tb:
+        import traceback
+        traceback.print_exc()
```

Comment:
This should probably go to stderr, so that it can be redirected separately for more convenient debugging in case the test app has substantial amount of regular output.

Reply:
sounds good